src/components/ModelizerDrawView.vue

```diff
@@ -157,15 +103,28 @@ async function updatePluginsAndTemplates() {
     return;
   }
 
-  drawComponents();
-
-  await getTemplatesByType('component', data.plugin.data.name)
-    .then((response) => {
+  await Promise.allSettled([
```

Comment:
no need to await, you can just `return Promise.allSettled`

Reply:
review with @Zorin95670 , not to do.